examples/boilerplate/hello-world-touch-disabled/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
```

Comment:
Not sure if we need an example specific for just one flag.

Reply:
Addressed in commit 1d6930b61ac66d080bfe57eada841a032fd6e6e8